doc/create-cloud-resources.rst

```diff
@@ -0,0 +1,145 @@
+.. _create-cluster:
+
+Create your cloud resources
+===========================
+
+BinderHub is built on top of JupyterHub, which uses Kubernetes to manage
```

Comment:
"BinderHub is built to run on top of Kubernetes, a distributed cluster manager. It also uses a JupyterHub to do the actual launching of user servers and a docker registry to cache images. To create your own BinderHub, you'll first need to set up a properly configured Kubernetes Cluster on the cloud, and then configure the various components correctly"